apps/aurora-portal/package.json

```diff
@@ -16,6 +16,7 @@
     "prod": "NODE_ENV=production node dist/polaris-bff/server.js"
   },
   "dependencies": {
+    "@fastify/cookie": "^11.0.2",
```

Comment:
Don't you pin dependencies?
https://docs.renovatebot.com/dependency-pinning/#should-you-pin-your-javascript-dependencies